src/gosling-track/gosling-track.ts

```diff
@@ -63,6 +64,18 @@ interface GoslingTrackOption {
     theme: CompleteThemeDeep;
 }
 
+interface GoslingTile extends Tile {
+    mergedToAnotherTile: boolean;
+    goslingModels?: GoslingTrackModel[];
+    gos: Record<string, any> & {
+        tabularData?: Record<string, any>[];
+        raw?: Datum[];
+    };
+    tileData: TileData & {
+        tabularData?: Record<string, any>[];
+    };
+}
+
```

Comment:
We should really unify the use of `tileData`, `tileData.tabularData`, `gos.raw`, and `gos.tabularData`.

Reply:
We can remove the `gos` object and just use the `tileData`. I will do this in a separate PR.